reference-architecture/rhv-ansible/ocp-vars.yaml

```diff
@@ -45,6 +45,7 @@ root_ssh_key:
 deployment_type: openshift-enterprise
 openshift_vers: v3_5
 containerized: false
+console_port: 8443
```

Comment:
This only got added because the verification roles require it, and fail without it.